tools/internal_ci/linux/grpc_build_packages.sh

```diff
@@ -30,6 +30,7 @@ set -ex
 # where they can be accessed from within a docker container that builds
 # the packages
 mv ${KOKORO_GFILE_DIR}/github/grpc/artifacts input_artifacts || true
+chmod +x input_artifacts/protoc*/* || true
```

Comment:
nit: outside of the scope of this PR, but is this `chmod` redundant?

Reply:
It might be but it is a good idea to have both for robustness: this preserves compatibility with the old Jenkins flow and other scenarios in the future (and other scripts that may have not changed for this). The other one is good to have so that the Ruby build script is robust against this sort of failure.